source/plugin/permissions.rst

```diff
@@ -153,9 +153,9 @@ Otherwise it will be inherited from any parent ``Subject``. It does not matter w
 
 If neither the subject itself nor any parent subjects grant or deny a permission then it will be inherited from the
 default ``Subjects``. Each ``SubjectCollection`` defines its own defaults. The global and weakest default subject can be
-obtained from the ``PermissionService``. It is recommended that plugins define their own permissions to the default's
-transient ``SubjectData`` during every server start-up. This allows the server owner to overwrite the defaults defined
-by plugins according to his needs using the default's persistent ``SubjectData``.
+obtained from the ``PermissionService``. It is recommended (but not mandatory) that plugins define their own permissions
```

Comment:
I'm heavily against even having the word recommended even be here.
```
It is possible for plugins to define their own permissions on the default transient ``SubjectData`` during every server start-up, however care should be taken not to introduce unnecessary complexity.

This allows the server owner to overwrite the defaults defined by plugins according to their needs using the default's persistent ``SubjectData``.

This however requires the server owner to have knowledge of the default permissions your plugin is granting in the first place, so must be carefully documented.

It is recommended that you do not have any default permissions assigned, unless you are very sure that it will reduce the complexity of your plugin, most plugins should never need to do this.
```